pkgs/development/python-modules/libnacl/default.nix

```diff
@@ -1,33 +1,41 @@
-{ lib, stdenv, buildPythonPackage, fetchFromGitHub, pytest, libsodium }:
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, libsodium
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
   pname = "libnacl";
-  version = "1.7.1";
+  version = "1.7.2";
 
   src = fetchFromGitHub {
     owner = "saltstack";
     repo = pname;
     rev = "v${version}";
-    sha256 = "10rpim9lf0qd861a3miq8iqg8w87slqwqni7nq66h72jdk130axg";
+    sha256 = "sha256-nttR9PQimhqd2pByJ5IJzJ4RmSI4y7lcX7a7jcK+vqc=";
   };
 
-  checkInputs = [ pytest ];
   propagatedBuildInputs = [ libsodium ];
```

Comment:
Shouldn't it sufficient for this to be in `buildInputs`?

Reply:
Yes, it's not needed during runtime.

```bash
ew/pr-113050-2/results/python38Packages.libnacl/lib/python3.8/site-packages]$ python
Python 3.7.9 (default, Aug 15 2020, 06:36:35) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import libnacl.public
>>> fred = libnacl.public.SecretKey()
>>> raw_sk = fred.sk
>>> print(raw_sk)
b'\x8a>\xfa\xfe4\xa7M \x06}D\xea@\xa4\x02M1w \xda\xb0L\x8f\x8dR\xc47\x01\x00\nxM'
```